packages/app/src/nativeBridge/modules/logsModule/index.ts

```diff
@@ -39,8 +39,31 @@ const bridgeState: {
   },
 };
 
+const READ_TIMEOUT_MS = 60000;
```

Comment:
is it more accurate to call it "timeout" or "idle"? it's not that something failed in IO, but simply no actual logs being written, right?

Reply:
yeah it's a gray area -- from the perspective of the match we were expecting to find more logs but didn't (waiting for the activity to end timed out); from the perspective of reading the disk nothing is wrong we just don't have more data (the combat log writer is idle)